### ISSUE TITLE
Which DID Methods are primarily funded by, will be controlled by, or primarily to the benefit of the US Government (US military complex, DHS, FBI, CIA, CBP, immigration, etc.)?

### ISSUE THREAD

**Issue by mwherman2000** [decentralized-identity/did-methods#15]
Which DID Methods are primarily funded by, will be controlled by, or primarily to the benefit of the US Government (US military complex, DHS, FBI, CIA, CBP, immigration, etc.)?

@peacekeeper @kimdhamilton

**Comment by brianorwhatever** [comment]
I also don't think this is of particular relevance. This group should be focused on technical merit, not funding mechanisms.

**Comment by mwherman2000** [comment]
For example, if there is a DID Method that is very specific to, for example, DHS/CBP with little to no imagined reusability, it's priority IMO should be ranked quite low.

Universal DID Methods on the other hand should be ranked quite high (within their cluster).

**Comment by brianorwhatever** [comment]
yes, they should be. Not because of where the money that funded them came from but because a universal DID method is more generally useful as a DID method.
